src/main/controllers/TribunalRespondToOrderController.ts

```diff
@@ -70,10 +72,17 @@ export default class TribunalRespondToOrderController {
       return res.redirect(returnSafeRedirectUrl(req, redirectUrl, logger));
     }
     req.session.errors = [];
+
+    const ECCRedirectUrl =
```

Comment:
This is slightly confusing as the return value for this may be the next page for other types of notifications, not just ECC. I would move all of this logic to get the redirectUrl out to a helper function. The first check you should do is for supporting material, then check if you need to skip the Rule92 page

Reply:
Refactored